README.md

```diff
@@ -425,16 +425,25 @@ Install Galaxy as per the current production server best practices:
       become: yes
     # Install with:
     #   % ansible-galaxy install natefoo.postgresql_objects
-    - role: natefoo.postgresql_objects
+    - role: galaxyproject.postgresql_objects
       become: yes
       become_user: postgres
     - role: galaxyproject.galaxy
   handlers:
-    - name: Restart Galaxy
-      supervisorctl:
-        name: galaxy
-        state: restarted
-      listen: restart galaxy
+    - name: Galaxy gravity restart
+      command: "/usr/local/bin/galaxyctl graceful"
```

Comment:
```suggestion
      command: "{{ galaxy_gravity_command }} graceful"
```

Reply:
I think this sadly doesn't work because `galaxy_gravity_command` is a default within the `galaxy` role, so it's not available in the playbook. But `{{ galaxy_venv_dir }}/bin/galaxyctl` might be a bit better?

Reply:
Ah I knew I was using that based on seeing it from somewhere: the built in handlers We have two handlers defined, that call this already (`handlers/gravity_23.0.yml`)

So these handlers as defined in the example are no longer necessary.

Reply:
Ah right! Those handlers weren't working for me, but I now see that may be due to https://github.com/galaxyproject/ansible-galaxy/issues/202 -- so probably that issue should be fixed, and the handlers should not be defined in the Readme. :)